db/model/model.js

```diff
@@ -37,3 +37,17 @@ exports.getArticle = () => {
     return rows;
   });
 };
+
+exports.getCommentsByArticleIds = (article_id) => {
+  let queryStr = "SELECT * FROM comments WHERE article_id = $1";
+  const args = [];
+  args.push(article_id);
+
+  return db.query(queryStr, args).then((comments) => {
+    console.log(comments.rows);
+    if (!comments.rows.length) {
```

Comment:
You will need to check the reason for the query returning empty rows:
- article exists but has no comments.
- article does not exist.

You will need to check whether the article exists in order to determine this.